docs/wikis/wikis.md

```diff
@@ -0,0 +1,67 @@
+# Wikis
+
+## Setup
```

Comment:
This should be in the Gitbook readme. (From my point of view) 
Please discuss with @jaskiratsingh2000 @vaibhavdaren

Reply:
@vaibhavdaren @jaskiratsingh2000 Please share your opinion on this. Thanks

Reply:
move this to more accesible place